packages/components/.eslintrc.cjs

```diff
@@ -36,7 +36,8 @@ module.exports = {
         'plugin:@typescript-eslint/recommended',
       ],
       rules: {
-        // Add any custom rules here
+        // ember doesn't support template only components that have type signatures so we need to use empty component classes
+        'ember/no-empty-glimmer-component-classes': 'off',
```

Comment:
blocker: we don't have to turn this off in `packages/component`; we should fix this by updating the (8?) instances where we have empty backing classes with `TemplateOnlyComponent`s

Reply:
This is needed for the `gts` conversion. There is currently no way for the gts components to have types and only a template. The showcase text components are examples of this ([Shw::Text::Body](https://github.com/hashicorp/design-system/blob/main/showcase/app/components/shw/text/body.gts)).

I can make it more specific so the rule is only turned off for `.gts` files and not `.ts` files too.

Reply:
ok, I understand this may be a must for showcase (a link to the issue, if possible, would be of help so that we keep track of this), but is it required for the components as well? (as we don't have `.gts` components in our library)

Reply:
from where I stand, this rule found quite a few instances in `components` where we've not been very thorough in how we converted our components to TypeScript, so I'd be keen to correct them

Reply:
I corrected the failures in the components and. removed the override!